Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -468,7 +468,15 @@ private Process GetProcessById(int id)
         {
             try
             {
-                return id <= 0 ? Process.GetCurrentProcess() : Process.GetProcessById(id);
+                switch (id)
+                {
+                    case 0:
+                        return Process.GetCurrentProcess();
+                    case -1:
+                        return Process.GetProcessesByName("w3wp").FirstOrDefault(s => s.Id != Process.GetCurrentProcess().Id) ?? Process.GetCurrentProcess();
```

Comment:
Do we want to get this change in? I needed it in the private kudu because the script host loads a native dll for edge js that get's locked and I can't update it.